recipes/rocksdb/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "rocksdb" %}
+{% set version = "6.2.2" %}
+
+package:
+  name: rocksdb
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebook/rocksdb/archive/v{{ version }}.tar.gz
+  sha256: 3e7365cb2a35982e95e5e5dd0b3352dc78573193dafca02788572318c38483fb
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake >=3.5.1,<4
+
+  host:
+    - gflags >=2.2.2,<3
+    - jemalloc >=4.5.0,<5
+    - snappy >=1.1.7,<2
+  run:
+    - {{ pin_compatible("jemalloc", exact=True) }}
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/librocksdb.a
+    - test -f ${PREFIX}/lib/librocksdb.dylib  # [osx]
+    - test -f ${PREFIX}/lib/librocksdb.so     # [linux]
```

Comment:
If the intend is to also provide development headers, I would add:
```
test -f $PREFIX/include/rocksdb/db.h
```

Reply:
added the .h test.